test/e2e/cli-install-uninstall/02-errors.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kudo.dev/v1alpha1
+kind: Instance
+metadata:
+  labels:
+    kudo.dev/operator: cli-install-operator
+  name: cli-install
```

Comment:
I think here it's ok just to assert apiversion + kind + metadata.name right? Having the yamls too verbose then makes it harder to adjust test when something in API change :) So I am always trying to assert the minimum that makes sense...

Reply:
I minimized the assertions.